example_1/package.xml

```diff
@@ -19,6 +19,8 @@
   <depend>rclcpp</depend>
   <depend>rclcpp_lifecycle</depend>
 
+  <exec_depend>ros2_controllers_test_nodes</exec_depend>
```

Comment:
Doesn't all packages automatically depend on this?

Reply:
what do you mean by automatically?
I would say that lots of dependencies are actually missing here, `joint_trajectory_controller` etc. Or would you include this in the ros2_control_demo meta package? (which has to be updated as well)

Reply:
yes, these dependencies are missing for all examples. I can add them to all examples once this one is merged into new structure branch?

Reply:
yes please!